sdk/cloudmachine/Azure.Projects.OpenAI/api/Azure.Projects.OpenAI.net8.0.cs

```diff
@@ -1,5 +1,22 @@
+namespace Azure.AI.Models
+{
+    public partial class ModelsClient
+    {
+        protected ModelsClient() { }
+        public ModelsClient(System.Uri endpoint, Azure.Core.TokenCredential credential) { }
+        public ModelsClient(System.Uri endpoint, Azure.Core.TokenCredential credential, OpenAI.OpenAIClientOptions options) { }
+        public ModelsClient(System.Uri endpoint, System.ClientModel.ApiKeyCredential credential) { }
+        public ModelsClient(System.Uri endpoint, System.ClientModel.ApiKeyCredential credential, OpenAI.OpenAIClientOptions options) { }
+        public OpenAI.Chat.ChatClient GetChatClient(string model) { throw null; }
```

Comment:
should we have a `GetModels` API?

Reply:
what would it do?

Reply:
Give you a list of models that have been registered so code could know which ones it can request.

Reply:
which endpoint/operation are you talking about?

Reply:
well,  let's discuss this as a separate feature.